isis/CMakeLists.txt

```diff
@@ -28,15 +30,15 @@ set(PACKAGE            "ISIS")
 set(PACKAGE_NAME       "USGS ISIS")
 
 # Version number
-set(VERSION            "3.5.00.0")
+set(VERSION            "3.6.00.0")
 set(PACKAGE_VERSION    ${VERSION})
 
 # Full name and version number
 set(PACKAGE_STRING     "${PACKAGE_NAME} ${VERSION}")
 
 # Other release information
-set(VERSION_DATE              "2017-04-24")
-set(THIRD_PARTY_LIBS_VERSION  "v007")
+set(VERSION_DATE              "2018-11-09")
+set(THIRD_PARTY_LIBS_VERSION  "v008")
```

Comment:
Why did we need this line in here exactly...?

Reply:
I am not sure. I am going to look into why this change was made.

Reply:
Ian and Makayla updated this when they put in gtest.